misc/ansible/screenly.yml

```diff
@@ -0,0 +1,38 @@
+- hosts: all
+  user: pi
+  vars:
+    screenly_systemd_units:
+      - X.service
+      - matchbox.service
+      - screenly-viewer.service
+      - screenly-web.service
+
+  tasks:
+    - name: Ensure screenly config dir exists
+      file: path=/home/pi/.screenly state=directory
+
+    - name: Copy screenly default config
+      copy: src=/home/pi/screenly/misc/screenly.conf dest=/home/pi/.screenly/screenly.conf
```

Comment:
This looks scary. Are you sure this will never override the file? Shouldn't we perhaps use the [stat](https://docs.ansible.com/ansible/stat_module.html) module?